static-site/posts/updates/2019.3.11-update.udon

```diff
@@ -0,0 +1,31 @@
+:-  :~  date/'~2019.3.20'
+        title/'~2019.3.11 Update'
+        preview/'A short update from the Tlon team for the week of ~2019.3.11.'
+        author/'~hidrel-fabtel'
+        type/'post'
+    ==
+;>
+
+# ~2019.3.11 Update
+
+Here’s a short update from the team for the week of `~2019.3.11`:
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.3.11-update-1.jpg"(width "100%");
+
+## `@p` Bug
+
+The `@p` bug [is nearly solved](https://github.com/urbit/arvo/pull/1110), thanks to [@max19](https://github.com/urbit/arvo/issues/1105#issuecomment-472585937) and *Jared* `~nidsut-tomdun`.
+
+## Team 
+
+*Ted* `~rovnys-ricfer`, *Philip* `~wicdev-wisryt`, and *Joe* `~master-morzod` developed a proposal for formalizing the semantics around state persistence in a way that enables parallelism, event replay, and a new model for error handling. *Joe* `~master-morzod` worked on integrating new `%eyre`. *Elliot* `~ponnys-podfer` worked on an elaborate proposal for resolving the `@p` bug involving constraint solvers, which turned out to be unnecessary, but was quite impressive nonetheless!
+
+*David* `~rapfyr-diglyt`, *Robert* `~lodleb-ritrul`, and 23 students completed the first week of Hoon School. `~hadrud-lodsef` implemented automation for city onboarding. In the next few weeks we're looking forward to automating Urbit support, the Hoon School queue, and contributor onboarding.
```

Comment:
Did you forget to mention Will, or did he ask not to have his real name mentioned? Also, I don't think anyone will understand what "city onboarding" means, so maybe quickly explain it somehow.